### PR TITLE
BRICK-1120 attach /var/amobee to tmp location

### PR DESCRIPTION
**Pull request by ldoty**

### CODE REVIEW

File: bricks/config.rb [hunk 1]
```diff
@@ -127,7 +127,7 @@ def custom_config(config)
 # or, to map host folders to guest folders of the same name,
 # $shared_folders = Hash[*['/home/foo/app1', '/home/foo/app2'].map{|d| [d, d]}.flatten]
 #$shared_folders = {}
-$shared_folders = {'/var/log/amobee' => '/var/log/amobee', '/etc/amobee' => '/etc/amobee'}
+$shared_folders = {'/var/log/amobee' => '/var/log/amobee', '/etc/amobee' => '/etc/amobee',  '/var/amobee' => '/tmp/var/amobee'}
```
Comment (mordr) [inline bricks/config.rb]: I thought we might want to retain the data in there folders. Attaching to `/tmp/**` means we can lose that data anytime ?

**Comment by mordr** [comment]
done with first pass.

**Comment by ldoty** [comment]
I got an e-mail for a comment here I cannot see anymore.

Ragib:
I thought we might want to retain the data in there folders. Attaching to /tmp/** means we can lose that data anytime ?

Good point, even though we are migrating this data into /var/amobee. Using the tmp is a probably a bad idea (I think it gets removed on startup). I will move it to a different directory.

**Review by mordr — APPROVED** [review]
lgtm

**Review by ahawtho — APPROVED** [review]

**Pull request merged**